pkg/swagger/typewalker.go

```diff
@@ -141,6 +141,9 @@ func (tw *typeWalker) schemaFromType(t types.Type, deps map[*types.Named]struct{
 
 				properties := tw.schemaFromType(field.Type(), deps)
 				properties.Description = strings.Trim(node.Doc.Text(), "\n")
+				if field.Name() == "WorkerProfilesStatus" {
+					properties.ReadOnly = true
+				}
```

Comment:
does ARM prevent the user passing in values for this if we set the swagger specification to ReadOnly?  If not, we should do a server-side check to ensure the user can't actually patch this.

Reply:
Updated _openshiftcluster_validatestatic.go_ to validate _workerProfilesStatus_ filed is nil in the input request. This gets called during preflightvalidation.